intro-HPC/glossary.tex

```diff
@@ -161,5 +161,5 @@
 \newglossaryentry{Walltime}
 {
   name={Walltime},
-  description={Walltime is the length of time specified in the job-script for which the job will run on a batch system},
+  description={Walltime is the length of time specified in the job-script for which the job will run on a batch system, this can be seen as the time that passed as measured by a normal clock that is on your wall. This is a computational resource},
```

Comment:
Walltime is basically the time that passes as measured by your favourite wall mounted clock or your wrist watch. In the HPC context it either means the maximal amount of time a job will be allowed to run on a batch system or, after a job has completed, the actual time it ran.